frontend/src/components/App/RouteSwitcher.tsx

```diff
@@ -21,7 +24,7 @@ export default function RouteSwitcher() {
             <Route
               path={route.path}
               component={() => (
-                <PageTitle title={route.name ? route.name : route.sidebar}>
+                <PageTitle title={t(route.name ? route.name : route.sidebar)}>
```

Comment:
This is where it is translated.

Reply:
I was thinking there's some static analysis on i18next-parser that gets the strings for translation, and that this was not recommended, but I am not 100% sure this is the case.
Maybe the one we discussed works in this example but we'll see lots of missing messages for the ones we don't have a translation for?

Reply:
Ah right... good point.

Yes, I'll check that. 
(Maybe it needs to have a separate PageTitle in each route so the tool can pick up the `t('')`.)